code/modules/reagents/reagent_containers/food.dm

```diff
@@ -120,3 +120,43 @@
 /obj/item/weapon/reagent_containers/food/snacks/tortillachip/New()
 	..()
 	reagents.add_reagent(NUTRIMENT,1)
+
+/obj/item/weapon/poutineocean
+	name = "poutine ocean"
+	desc = "This enormous, horrific cheese wheel is filled to the brim with poutine. You'll probably never reach the bottom."
+	icon = 'icons/obj/food_huge.dmi'
+	icon_state = "poutineocean"
+	pixel_x = -16
+	pixel_y = -8
+	w_class = W_CLASS_GIANT
+
+/obj/item/weapon/poutineocean/attack_hand(mob/user)
+	to_chat(user, "<span class='warning'>You need a plate!</span>")
+
+/obj/item/weapon/poutineocean/attackby(obj/item/W, mob/user as mob)
+	if (istype(W, /obj/item/trash/plate))
+		qdel(W)
```

Comment:
Could do `if(user.drop_item(W))`, which would also cause them to drop the plate if they can.